zuul.d/pipelines.yaml

```diff
@@ -114,3 +114,42 @@
       sqlreporter:
     failure:
       sqlreporter:
+
+- pipeline:
+    name: third-party-check
+    description: |
+      Newly uploaded patchsets to projects that are external to OpenStack
+      enter this pipeline to receive an initial +/-1 Verified vote.
+    success-message: Build succeeded (third-party-check pipeline).
+    # TODO(mordred) We should write a document for non-OpenStack developers
+    failure-message: |
+      Build failed (third-party-check pipeline) integration testing with
+      OpenStack. For information on how to proceed, see
+      http://docs.openstack.org/infra/manual/developers.html#automated-testing
+    manager: independent
+    trigger:
+      github:
+        - event: pull_request
+          action:
+            - opened
+            - changed
+            - reopened
+        - event: pull_request
+          action: comment
+          comment: (?i)^\s*recheck\s*$
+    start:
+      github.com:
```

Comment:
I've changed this from the original to match the `check` pipelines `start` at the top of this file